reference/library/src/morpheus-master/sass/modules/tool/_base.scss

```diff
@@ -59,8 +59,7 @@
 		float: right;
 		display: block;
 		position: relative;
-		margin: 0px 0 -20px 0;
-		top: 9px;
+		margin: -1em 0 0 0;
```

Comment:
Could you add this modification to the tool itself? in `sass/modules/tool/iframe` under the namespace `#{$namespace}sakai-iframe`

This class is used all along sakai and could be dangerous to modify for all.